where-to-travel/src/main/webapp/index.html

```diff
@@ -1,28 +1,155 @@
-              <input type="text" name="hours" id="hrs">
-              <label>Minutes</label>
-              <input type="text" name="minutes" id="mnts">
+    <div class="bmd-layout-container bmd-drawer-f-l bmd-drawer-overlay">
+      <header class="bmd-layout-header">
+        <div class="navbar navbar-dark bg-faded">
+          <button class="navbar-toggler" type="button" data-toggle="drawer" data-target="#dw-s2">
+            <span class="sr-only">Toggle drawer</span>
+            <span class="btn text-color" style="font-size: 1.2em; color: #049688">
+              &#9776;
+            </span>
+          </button>
+          <ul class="nav navbar-nav">
+            <li class="nav-item">
+                <img onclick="showInfoModal()" class="btn btn-icon" src="icons/help.svg">
+              <a class="btn btn-outline-primary" style="text-align: center" href="login.html">Login</a>
+              <span id="nav-text">or</span>
+              <a class="btn btn-outline-primary" href="signup.html">Sign up</a>
+            </li>
+          </ul>
+        </div>
+      </header>
+      <div id="dw-s2" class="bmd-layout-drawer bg-faded">
+        <header>
+          <a class="navbar-brand">Search Settings</a>
+        </header>
+        <div id="accordion">
+          <div class="card">
+            <div class="card-header" id="headingOne">
+              <h1 class="mb-0">
+                <button class="btn btn-link collapsed text-color" data-toggle="collapse" data-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
+                  Travel Time
+                </button>
+              </h1>
+            </div>
+            <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion">
+              <div class="card-body">
+                <form class="form-inline form-mixin">
+                  <div class="form-group fg-mixin">
+                    <label for="hours" class="bmd-label-floating text-color">HH</label>
+                    <input id="hrs" style="width: 100%; margin-right:0px;" type="text" class="form-control">
+                  </div>
+                  <div id="colon" class="form-group bmd-label-static">
+                    :
+                  </div>
+                  <div class="form-group fg-mixin">
+                    <label for="minutes" class="bmd-label-floating text-color">MM</label>
+                    <input id="mnts" style="width: 100%; margin-right:0px;" type="text" class="form-control">
+                  </div>
+                  </form>
+              </div>
             </div>
-            <button id="submit-id" type="button">Search</button>
-          </form>
+            <div class="card">
+              <div class="card-header" id="headingTwo">
+                <h1 class="mb-0">
+                  <button class="btn btn-link collapsed text-color" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
```

Comment:
This line also looks long, I realize I probably wrote it originally, but it would improve readability if it wasn't so long.

Reply:
fixed!